resources/views/account/partials/address-cards-popup.blade.php

```diff
@@ -30,6 +30,12 @@ class="w-full sm:min-w-[350px]"
                     <x-rapidez-ct::button.link v-else v-on:click="userAddress.default_billing=false;mutate()">
                         @lang('Deselect as billing')
                     </x-rapidez-ct::button.link>
+                    <graphql-mutation query="mutation deleteCustomerAddress($id: Int!){ deleteCustomerAddress ( id: $id ) }" :variables="{ id: userAddress.id }" v-slot="{ mutate }">
+                        <x-rapidez-ct::button.link v-if="!userAddress.default_billing && !userAddress.default_shipping" v-on:click="data.customer.addresses.splice(addressIndex,1);mutate()">
```

Comment:
It would be nice to have a confirm popup when a user deleting there address.

Reply:
I think if we wanna do this we should build a standard way to have confirmation dialogs, and IMO that falls outside of the scope of this for now.